src/dialog.tsx

```diff
@@ -10,14 +10,16 @@ import {isWebViewBridgeAvailable, nativeConfirm, nativeAlert} from '@tef-novum/w
 import ThemeContext from './theme-context';
 import {useTheme, useScreenSize} from './hooks';
 import ButtonLayout from './button-layout';
-import {Text5, Text3} from './text';
+import {Text5, Text4, Text3} from './text';
 import {ESC} from './utils/key-codes';
 import Box from './box';
 import {isOldChrome, isRunningAcceptanceTest} from './utils/platform';
 import {useSetModalState} from './modal-context-provider';
+import {pxToRem} from './utils/css';
 import Stack from './stack';
 
 import type {Theme} from './theme';
+import type {RendersNullableElement} from './utils/types';
 
```

Comment:
I found a weird space above the title in confirm and alerts in mobile 
![image](https://user-images.githubusercontent.com/6722153/196685431-45b45024-6ba6-4d57-8b27-045bc0c75e6c.png)

in specs 
![image](https://user-images.githubusercontent.com/6722153/196685574-479cd051-4bab-490b-a40a-177afb8ea14e.png)

I think the cause of this is when the modal doesn't have dismiss, you don't need to include this extra space